bedrock-entry.sh

```diff
@@ -13,12 +13,15 @@ fi
 set -e
 
 case ${VERSION} in
-  1.11|PREVIOUS)
+  1.11)
     VERSION=1.11.4.2
     ;;
-  1.12)
+  1.12|PREVIOUS)
     VERSION=1.12.0.28
```

Comment:
I wish I had noted the newest 1.12.1 version they released...that would have been good to set here for 1.12.